Packs/CloudConvert/ReleaseNotes/1_0_9.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### CloudConvert - Convert File
+
+- Fixed an issue where the playbook contained unhandled script condition branches.
```

Comment:
lets be more clear about what the change was here. Added "x" branch in the playbook, or something like this. Same for the other rn

Reply:
Only in the Service desk plus we've actually changed something, for other playbooks it's just "cosmetics"